gc/base/SparseAddressOrderedFixedSizeDataPool.hpp

```diff
@@ -103,6 +103,7 @@ class MM_SparseAddressOrderedFixedSizeDataPool : public MM_BaseVirtual
 	static MM_SparseAddressOrderedFixedSizeDataPool *newInstance(MM_EnvironmentBase *env, void *sparseHeapBase, uintptr_t sparseDataPoolSize);
 	virtual void kill(MM_EnvironmentBase *env);
 
+	J9HashTable *getObjectToSparseDataTable() { return _objectToSparseDataTable; }
```

Comment:
formatting not consistent with other methods; also, method description is missing
```suggestion
	J9HashTable *getObjectToSparseDataTable()
	{
		return _objectToSparseDataTable;
	}
```